### PR TITLE
Add assets for 'LOKINU-bc11a1'

### PR DESCRIPTION
**Pull request by Simlaw**

### CODE REVIEW

**Comment by Simlaw** [comment]
We've been waiting for your approval to launch, and our community is waiting as well. We will have the conditions you require very quickly after the launch. But for now, we cannot have more holders without the launch.

**Review by paulmultiversx — APPROVED** [review]

**Pull request merged**
